web/views.py

```
@@ -4807,58 +4787,81 @@ def donation_webhook(request):
+        elif event.type == "invoice.payment_failed":
+            invoice = event.data.object
+            logger.info("Processing failed invoice payment: %s", invoice.id)
+            handle_invoice_failed(invoice)
+
+        return HttpResponse(status=200)
+    except Exception:
+        logger.exception("Error processing webhook %s", event.type)
+        return HttpResponse(status=500)
 
 
 def handle_successful_donation_payment(payment_intent):
```

Ruff (0.8.2)

Missing return type annotation for public function `handle_successful_donation_payment`

Add return type annotation: `None`

(ANN201)